numpy/core/src/umath/matmul.c.src

```diff
@@ -0,0 +1,365 @@
+/* -*- c -*- */
+
+#define _UMATHMODULE
+#define _MULTIARRAYMODULE
```

Comment:
Is this the only file that defines both?

Reply:
`loops.c.src` defines both